discovery-provider/src/queries/get_track_history.py

```diff
@@ -0,0 +1,59 @@
+from src.models import Play, Track
+from src.utils import helpers
+from src.utils.db_session import get_db_read_replica
+from src.queries import response_name_constants
+from src.queries.query_helpers import (
+    add_query_pagination,
+    populate_track_metadata,
+    get_users_by_id,
+    get_users_ids,
+)
+
+
+def get_track_history(args):
```

Comment:
since we have types in discovery now, this would be nice to be

```
class GetTrackHistoryArgs(TypedDict):
  current_user_id: int
  limit: int
  offset: int
  filter_deleted: bool

def get_track_history(args: GetTrackHistoryArgs):
```

Reply:
Updated